docs: add details on how to be elected as core contributor (#16)
NodeSecure/Governance: README.md

  </tr>
</table>

<details>

> <summary>How to be elected core contributor ?</summary>

Reviewer: ```suggestion
<summary>How to be elected core contributor?</summary>
```

> These are contributors who have **proven** their importance and impact through **multiple contributions** to one or more projects of the organisation.

Reviewer 2: ```suggestion
These are contributors who have **proven** their importance and impact through **multiple contributions** to one or more projects of the organization.
```

> It's not **just about code**, participating in the **life** of the project is just as important (discord, meetings ..). Developers who are champion or maintainer of at least one project have high probability to be elected to the core team.

Reviewer: ```suggestion
Contributing is not **just about coding**, it can also be participating in the **life** of the project which is just as important e.g. discord, meetings, branding, etc. Developers who are champions or maintainers of at least one project have a high probability to be elected to the core team.
```

> Contributing is not **only about coding**, it can also be participating in the **life** of the project which is just as important e.g. discord, meetings, branding, etc. Developers who are champions or maintainers of at least one project have a high probability to be elected to the core team.

Reviewer 2: ```suggestion
Contributing is not **only about coding**, but it can also be participating in the **life** of the project, which is just as important e.g. discord, meetings, branding, etc. Developers who are champions or maintainers of at least one project have a high probability of being elected to the core team.
```
